src/server/electron.ts

```diff
@@ -125,17 +115,16 @@ export class ElectronApplication extends EventEmitter {
 
   async close() {
     const closed = this.waitForEvent(ElectronEvents.ElectronApplication.Close);
-    await this.evaluate(({ app }) => app.quit());
+    await this._nodeElectronHandle!.evaluate(({ app }) => app.quit());
     this._nodeConnection.close();
     await closed;
   }
 
-  async waitForEvent(event: string, optionsOrPredicate: types.WaitForEventOptions = {}): Promise<any> {
-    const options = typeof optionsOrPredicate === 'function' ? { predicate: optionsOrPredicate } : optionsOrPredicate;
-    const progressController = new ProgressController(this._timeoutSettings.timeout(options));
+  async waitForEvent(event: string, predicate?: Function): Promise<any> {
```

Comment:
I'm surprised to see waitForEvent still here. It got put on the clientside elsewhere.

Reply:
Maybe it should be _waitForEvent?

Reply:
Yeah, I can make it private in a follow up!